core/sched.c

```diff
@@ -74,8 +74,10 @@ FORCE_USED_SECTION
 const uint8_t _tcb_name_offset = offsetof(thread_t, name);
 #endif
 
+#ifdef MODULE_SCHED_CB
+static void (*sched_cb) (uint32_t value_1, uint32_t value_2) = NULL;
```

Comment:
Better rename to `active_thread`, `next_thread` here too.

Reply:
You are right, I had though of that. I didn't want to change right away because `sched_cb` has been in for 8 years, although I couldn't find a time where is was used. I wasn't sure if it would be ok to change it.

@kaspar030 any reason for kipping`sched_cb` as it is? Do you know if it was added for a specific use case that is not present in RIOT code (couldn't find any reference to `sched_register_cb` threw git grep.)

Reply:
I went ahead and changed it...